### DISCUSSION TITLE
<C-n> and <C-p> to navigate menu

### DISCUSSION THREAD

**Discussion by jingchengx** [ahrm/sioyek#1103] [General]
I just moved to the master branch from the 2.0.0 release and I can no longer use `<C-n>` and `<C-p>` to navigate the menu.  Is this intended?  Is there some way to re-enable it?

**Comment by ahrm** [comment]
We have added the ability to customize such commands in the development branch. I don't remember exactly but maybe you need to add something like `emacs_mode_menus 1` to activate it in the main branch?

**Reply by jingchengx** [reply]
Thanks!  Looking at the code there does seem to be such an option but it doesn't do anything in the current build.
It seems the `emacs_mode_menus` should enable `C-v` and `M-v` as page down and up, and the `C-n` as arrow down should always work?  In any case `C-n` doesn't do anything in the menu regardless of the setting.
